pkg/webhook/validate.go

```diff
@@ -29,8 +29,9 @@ const (
 )
 
 var (
-	nodesSelected     bool
-	interfaceSelected bool
+	nodesSelected          bool
+	interfaceSelected      bool
+	nodeInterfaceErrorList = make(map[string][]string)
```

Comment:
can we avoid global variables ?

personally not a fan of the existing ones, id prefer to avoid adding more (generally would be great if we got rid of those global vars... )

`validatePolicyForNodeStateAndPolicy` can be modified to accept this map as input. in 
`dynamicValidateSriovNetworkNodePolicy` we can define it and pass it to `validatePolicyForNodeStateAndPolicy`  then finally log its content.

WDYT ?

Reply:
FWIW, I agree w/ @adrianchiris